lib/gmock-1.7.0/gtest/test/gtest-test-part_test (1).cc

```diff
@@ -0,0 +1,208 @@
+// Copyright 2008 Google Inc.
```

Comment:
I'm not sure how this happened, but a couple of unrelated files got added in this commit.